ibis/backends/tests/snapshots/test_sql/test_cte_refs_in_topo_order/clickhouse/out.sql

```diff
@@ -0,0 +1,49 @@
+SELECT
```

Comment:
This isn't optimized, and we're not yet extracting subqueries in the clickhouse backend yet after the sqlglot refactor.